Common/Balance.swift

```diff
@@ -7,6 +7,30 @@
 
 import Foundation
 
+public enum OrderType: String {
+    case Buy = "Buy"
```

Comment:
String enum values are implicit. No need to specify explicitly unless the value is different from enum case name.